### PR TITLE
Update feature request template

### PR DESCRIPTION
**Pull request by bakerboy448**
git

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
 ---
 name: Feature request
 about: Suggest an idea for Sonarr
+title: ''
+labels: enhancement
```
Comment (markus101) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: I actually hate this about the bug template, half the bugs that get filed aren't bugs and new issues that need to be triaged get labeled as bugs straight away. IMO we shouldn't set a label so we can triage properly.

**Review by markus101 — CHANGES_REQUESTED** [review]
Personally I hate the auto tagging of issues.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (bakerboy448) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: I'll update the bug template as well then, incoming pushes

Might I suggest a Support label then so we don't have any non-labeled issues and can reserve non-labeled for items needing triage?  That'll allow us to tag the support GHIs accordingly.

Comment (markus101) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Not opposed to a label, but they get insta-closed most of the time.

**Pull request merged**